src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxIPAddressInformation.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    public class LinuxIPAddressInformation : IPAddressInformation
+    {
+        private IPAddress _address;
+
+        public LinuxIPAddressInformation(IPAddress address)
+        {
+            _address = address;
+        }
+
+        /// Gets the Internet Protocol (IP) address.
+        public override IPAddress Address { get { return _address; } }
+
+        /// Gets a bool value that indicates whether the Internet Protocol (IP) address is legal to appear in a Domain Name System (DNS) server database.
+        public override bool IsDnsEligible { get { throw new PlatformNotSupportedException(); } }
+
+        /// Gets a bool value that indicates whether the Internet Protocol (IP) address is transient.
+        public override bool IsTransient { get { throw new PlatformNotSupportedException(); } }
+    }
+}
```

Comment:
I think when I was merging these in BeyondCompare I wasn't able to tell that there were already newlines in place, and I ended up duplicating them. I'll go through and fix them.